feature_extractor/entity_helper.py

```diff
@@ -225,3 +234,31 @@ def _choose_professions(self):
     self._professions_feature = [
       one_hot_generator(N_ATK_TYPE, ATK_TYPE.index(prof)) for prof in profs
     ]
+
+  #####################################
+  # team-related helper functions
+  def pos_to_agent_id(self, member_pos):
+    return self._team_helper.agent_id(self._team_id, member_pos)
+
+  def is_pos_alive(self, member_pos):
+    return member_pos in self.member_location
+
+  def is_agent_in_team(self, agent_id):
+    return agent_id in self._team_agent_ids
+
+  def agent_id_to_pos(self, agent_id):
+    return self._team_helper.agent_position(agent_id)
+
+  def agent_team(self, agent_id):
+    return self._team_helper.team_and_position_for_agent[agent_id][0]
+
+  def agent(self, agent_id):
+    if agent_id not in self._entities:
+      return None
+
+    info = EntityState.parse_array(self._entities[agent_id].astype(np.int32))
+    # add level for using armors
+    info.level = max(getattr(info, skill + '_level')
```

Comment:
if you new-line before getattr, the alignment will be nicer

Reply:
yap, changed!